src/actions/dataActions.tsx

```diff
@@ -45,7 +45,7 @@ export function updateDatasets() {
            .catch(error => {
                dispatch(postMessage('error', error + ''));
            })
-           .finally(() => {
+           .then(() => {
```

Comment:
Please leave comment. why this is like so:

```suggestion
           // .finally() not implemented in Edge, see 
           // https://stackoverflow.com/questions/35999072/what-is-the-equivalent-of-bluebird-promise-finally-in-native-es6-promises
           .then(() => {
```